### ISSUE TITLE
Bug in tan

### ISSUE THREAD

**Issue by dpsanders** [JuliaIntervals/IntervalArithmetic.jl#21]
```jl
julia> tan(Interval(6.638314112824137, 8.38263151220128))
ERROR: ArgumentError: Must have a ≤ b to construct Interval(a, b).
Stacktrace:
 [1] IntervalArithmetic.Interval{Float64}(::Float64, ::Float64) at /Users/dpsanders/.julia/v0.6/IntervalArithmetic/src/intervals/intervals.jl:23
 [2] tan(::IntervalArithmetic.Interval{Float64}) at /Users/dpsanders/.julia/v0.6/IntervalArithmetic/src/intervals/trigonometric.jl:135
```
The checks according to quadrant are not correct.

**Comment by dpsanders** [comment]
Fixed by #22

**Comment by lbenet** [comment]
See this [comment](https://github.com/JuliaIntervals/IntervalArithmetic.jl/pull/22#issuecomment-307373482), which should have gone here.

**Comment by dpsanders** [comment]
Closed by #51.
